helm-charts/daskhub/values.yaml

```diff
@@ -1,4 +1,4 @@
-basehub:
```

Comment:
We are using the dask-gateway chart directly in binderhub, and I don't think that depends on the z2jh helm chart. So how should we copy these values over?